libr/anal/fcn.c

```diff
@@ -2188,7 +2188,7 @@ R_API int r_anal_fcn_cc(RAnal *anal, RAnalFunction *fcn) {
 	}
 
 	int result = E - N + (2 * P);
-	if (result < 1 && anal->verbose) {
+	if (result < 1 && anal && anal->verbose) {
 		eprintf ("Warning: CC = E(%d) - N(%d) + (2 * P(%d)) < 1 at 0x%08"PFMT64x"\n", E, N, P, fcn->addr);
```

Comment:
this is on purpose.. because if you get such wraning it means the function is wrongly analyzed and the analysis code must be fixed i would prefer to keep that warning